backend/frontend/views.py

```diff
@@ -672,6 +672,44 @@ def action_stop_card(request, card_id):
     )
 
 
+@login_required()
+def view_partial_users_list(request):
+    exclude = {
+        "is_superuser": True,
+    }
+    all_users = User.objects.filter(active=True).order_by("first_name", "last_name").exclude(**exclude)
```

Comment:
We should not display all users, we should display only the users that the current logged in user is allowed to see.

As things stand, any learner can log in and get a big list of users. We need to think about permissions all the time.